pkg/uzip/uzip_test.go

```diff
@@ -58,32 +63,102 @@ func TestFromZip(t *testing.T) {
 		t.Fatalf(`os.ReadFile("testdata/testFolder/subFolder/file4") = _, %v, want nil`, err)
 	}
 
-	if _, err := os.Stat(f1); err != nil {
-		t.Errorf(`os.Stat(%q) =_, %v, want nil`, f1, err)
+	var x []byte
+
+	x, err = ioutil.ReadFile(f1)
+	if err != nil {
```

Comment:
etc. etc. 
I know it's not always possible to follow the form:
"ioutil.ReadFile(%q): %v != nil"
but in this case it surely is :-)

Reply:
will do